api/DOMMatrixReadOnly.json

```
@@ -2068,6 +2068,54 @@
           }
         }
       },
+      "translate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMMatrixReadOnly/translate",
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
```

Opera 32 is based on Chrome 45, reportedly (https://help.opera.com/en/opera-version-history/#32 and https://forums.opera.com/topic/11499/opera-32-beta-released) so we can probably set both Operas to 32

```suggestion
              "version_added": "32"
```